README.md

```diff
@@ -806,7 +806,8 @@ See also [A comparison of operating systems written in Rust](https://github.com/
   * [Stavrospanakakis/is_ready](https://github.com/Stavrospanakakis/is_ready) - Wait for multiple services to become available ![Build](https://github.com/Stavrospanakakis/is_ready/actions/workflows/release.yml/badge.svg)
 * Heroku
   * [emk/heroku-buildpack-rust](https://github.com/emk/heroku-buildpack-rust) - A buildpack for Rust applications on Heroku
-* [release-plz](https://github.com/release-plz/release-plz) [[release-plz](https://crates.io/crates/release-plz)] - Release crates from CI, with changelog generation and semver check. [![build badge](https://github.com/release-plz/release-plz/workflows/CI/badge.svg)](https://github.com/release-plz/release-plz/actions)
+  * [release-plz](https://github.com/release-plz/release-plz) [[release-plz](https://crates.io/crates/release-plz)] - Release crates from CI, with changelog generation and semver check. [![build badge](https://github.com/release-plz/release-plz/workflows/CI/badge.svg)](https://github.com/release-plz/release-plz/actions)
+  * [biome](https://biomejs.dev/) - A toolchain for web projects, aimed to provide functionalities to maintain them. Biome offers formatter and linter, usable via CLI and LSP.
```

Comment:
* These are indented as if they're Heroku links, please un-indent
* Sorting will need fixing after that
* The link will fail, because it's not a Github one. Goto https://github.com/rust-unofficial/awesome-rust/blob/main/src/main.rs and add the url the bottom of `POPULARITY_OVERRIDES` flagging stars on https://github.com/biomejs/biome as the reason